Formula/pmd.rb

```diff
@@ -11,11 +11,41 @@ class Pmd < Formula
     sha256 "e5ed61e35b5c980b0e1bcf616bc2615f1c5310ad66ce8f4901ca225764e7e78c" => :mavericks
   end
 
+  depends_on :java => "1.8+"
   depends_on "maven" => :build
 
   def install
     ENV.java_cache
 
+    java_version = ENV["JAVA_HOME"][/((\d\.?)+)\.\d/, 1]
+    java_user_home = ENV["_JAVA_OPTIONS"][/-Duser.home=(.*) ?/, 1]
```

Comment:
Right. The point is to avoid hard coding a string in two places if it's only necessary to hard code it in one place.